src/lib.rs

```diff
@@ -5,6 +5,7 @@
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
 #![cfg_attr(feature = "nightly", feature(external_doc))]
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
That's maybe too premature since without the std feature there's no crate basically hehe. And if you use `--no-default-features` will make the compilation fail I think. I would remove it for now I think but feel free to tell what you think

Reply:
Makes sense. I added it as it might be useful later, but since it's only a single line, it won't matter much anyways.